<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cam_widget</class>
 <widget class="QWidget" name="cam_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:rgb(36, 31, 49);
color:rgb(255, 255, 255);
font-size:15px
}
            QPushButton {
                background-color: rgb(61, 56, 70);
                color:rgb(255, 255, 255);
                border: none;
                padding: 8px 16px;
                border-radius: 4px;
					font-size:20px
            }
            QPushButton:hover {
                background-color:rgb(192, 191, 188);
					color:black;
            }
QPushButton:disabled {
    background-color: rgb(222, 221, 218); /* Change the disabled color as desired */
}
QToolButton:pressed {
    background-color: rgb(153, 193, 241);  /* Change the pressed color as desired */
}
            QToolButton {
                background-color: rgb(53, 132, 228);
                color:black;
                border: none;
                padding: 8px 16px;
                border-radius: 4px;
            }
            QToolButton:hover {
                background-color:rgb(246, 245, 244);
					color:black;
            }
QToolButton:disabled {
    background-color: rgb(222, 221, 218); /* Change the disabled color as desired */
}
QPushButton:pressed {
    background-color: rgb(153, 193, 241);  /* Change the pressed color as desired */
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="camera_label">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPushButton" name="change_cam_button">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Change camera</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../icons/cam_switch.png</normaloff>../../icons/cam_switch.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
